response_operations_ui/templates/conversation-view/cv-reopen-form.html

```diff
@@ -1,4 +1,5 @@
 {% from "components/input/_macro.njk" import onsInput %}
```

Comment:
Is someone with the right endpoint and information able to reopen a conversation without the `messages.edit` permission?